client-sdk/ts-web/core/src/types.ts

```diff
@@ -2965,7 +2965,14 @@ export interface StakingStakeThreshold {
  */
 export interface StakingTakeEscrowEvent {
     owner: Uint8Array;
+    /**
+     * amount is the sum of amounts slashed from active and debonding escrow balances.
```

Comment:
you should add these in oasis-core. our reflect-go program will automatically transfer them here